fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -269,27 +305,43 @@ const CreateElection = () => {
                 ),
               )
             }
+            onRemove={() => setQuestions((prev) => prev.filter((_, id) => id !== idx))}
+            isRemovable={questions.length > 1}
             placeholder={STRINGS.election_create_question_placeholder}
           />
           <Text style={[Typography.paragraph, Typography.important]}>
             {STRINGS.election_create_ballot_options}
           </Text>
           <TextInputList
+            values={value.ballot_options}
             placeholder={STRINGS.election_create_option_placeholder}
             onChange={(ballot_options: string[]) =>
               setQuestions((prev) =>
                 prev.map((item, id) =>
                   id === idx
                     ? {
                         ...item,
-                        ballot_options: ballot_options,
+                        ballot_options: ballot_options.filter((option) => option !== ''),
```

Comment:
This likely causes a ballot option to disappear when it's text is deleted even if it is not the last one..? For `trimmedQuestions` you filter them a second time anyway 😄

Reply:
And I just noticed that this behavior was wrong. Now I need to filter them three times because we do not have a unique question id at this stage. :sweat_smile:

Reply:
If we do not remove the empty ones, we will have a new empty line for every character input, since our list is now a controlled component.

Reply:
Ah I see you're not removing it, then it is necessary yes. I thought you changed it (https://github.com/dedis/popstellar/pull/1456/files/dc095b6cec3c47c3966620b3a5ff2a1baa70577a#r1161113914) because I am pretty sure that now there is a bug if we delete all text in the not last ballot option (it is removed). Personally I would deem this weird/bad UX but it seems it is what you want?